mage_ai/frontend/oracle/components/FlyoutMenu/FlyoutMenuWrapper.tsx

```diff
@@ -8,6 +8,7 @@ type FlyoutMenuWrapperProps = {
 
 function FlyoutMenuWrapper({
```

Comment:
Create a separate file for `FlyoutMenuWrapper` with the code here duplicated, since I'm changing this component and want to avoid conflicts.

Reply:
done